src/jit/emitarm64.cpp

```diff
@@ -11350,7 +11350,7 @@ void emitter::emitDispIns(
                     const ssize_t mask8 = 0xFF;
                     for (unsigned b = 0; b < 8; b++)
                     {
-                        if (imm & (1 << b))
+                        if (imm & (size_t{1} << b))
```

Comment:
You are right, but because `imm` has type of `ssize_t` there is implicit conversion from `int` to `ssize_t` here.  It generates warning C4334 ('operator' : result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?), https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4334), that was disabled before.


The right thing to do here is to right:
`if (imm & static_cast<ssize_t>(1 << b)) `, but it also generates warnings, I think it is VC++ compiler issue and I have reported it.